### PR TITLE
Migrate block writer

### LINKED ISSUE #68: Migrate Schrodinger/write/block_writer
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

Refs: https://github.com/pingcap/schrodinger-test/tree/master/write/block_writer
Nemesis: randomly

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by mahjonp**
### What problem does this PR solve? 

close #68 

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cluster/k8s.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 	"regexp"
 
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```
Comment (mahjonp) [inline pkg/cluster/k8s.go]: need import client/auth on gke

File: pkg/nemesis/nemesis.go [hunk 2]
```diff
@@ -4,6 +4,7 @@ import (
 	"math/rand"
 	"os"
 
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```
[the remark this answers is not in the document]
Comment (mahjonp) [reply pkg/nemesis/nemesis.go]: ditto.

File: tests/block-writer/blockwriter.go [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+		blockCount:      0,
+		minSize:         128,
+		maxSize:         1024,
+		blockDataBuffer: make([]byte, 1024),
+		values:          make([]string, blockWriterBatchSize),
+	}
+}
+
+// Insert blockWriterBatchSize values in one SQL.
+// TODO: configure it from outside.
```
Comment (mapleFU) [inline tests/block-writer/blockwriter.go]: Is this todo still needed here?

Comment (mahjonp) [reply tests/block-writer/blockwriter.go]: addressed

**Pull request merged**